resources/puppetlabs/lein-ezbake/template/global/ext/fpm.rb

```diff
@@ -0,0 +1,376 @@
+#!/usr/bin/env ruby
```

Comment:
Is this a script you put together, or is it a generic fpm script that you've made lots of modifications to? I'm wondering about updating it for newer versions of fpm, if that's something we'll need to do in the future. If it's totally a custom one, and we don't need to update it based on an upstream script, just ignore me